ui/src-tauri/src/main.rs

```diff
@@ -522,8 +524,9 @@ async fn receiving_keys(server_store: State<'_, ServerStore>) -> Result<Vec<Stri
     if let Some(store) = &mut *server_store.lock().await {
         let keys = store
             .get_receiving_keys(Message {
+                // TODO: do we need to passing network?
```

Comment:
Do we have a case that need `receiving_keys` on ui?

Reply:
Hmm probably we need to ask Alex as this is based on his redesign too

Reply:
Signer_new_circuits is based on his redesign*

Reply:
We do have a case where we use receiving_keys on the UI, when the user successfully signs we want to display their zkAddress and allow them to copy it.

Reply:
current on ui, there are some fixed network as value, is that ok? Or do we need passing network as parameter?

Reply:
I mean we have 3 networks, hardcoding works best

Reply:
Bur we should probably change some function names like receiving keys and etc. Honestly this has taken this kuch time because if the lack of documentation around these things like sdk manta-rs signer stuff. I mean for one there is no way to know we are trying to deprecate scale_codec

Reply:
I agree renaming would be helpful, and I think manta-front-end doesn't yet support switching between networks because calamari doesn't have mantapay yet